docs/user/ppl/functions/conversion.rst

```diff
@@ -118,6 +118,58 @@ Use string in comparison operator example ::
     +------+-------+------+-------+------+------+------+
 
 
+
+TONUMBER
+--------
+
+Description
+>>>>>>>>>>>
+
+The following usage options are available, depending on the parameter types and the number of parameters.
+
+Usage: tonumber(string, [base]) converts the value in first argument to provided base type string in second argument. If second argument is not provided, then it converts to base 10 number representation.
```

Comment:
It's not converting 1st arg to provided base type in 2nd arg, right? The 2nd arg describe the base of 1st arg instead?